From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: "Max R. Carrara" <m.carrara@proxmox.com>
Date: Thu, 17 Jul 2025 17:20:08 +0200
Subject: [PATCH 61/61] mgr/zabbix: fix invalid escape sequences

Even though the `zabbix` module is deprecated and will be removed
soon [1], fix the warning regarding invalid escape sequences here
also, as it might look scary for some users.

[1]: https://github.com/ceph/ceph/pull/57299

Signed-off-by: Max R. Carrara <m.carrara@proxmox.com>
---
 src/pybind/mgr/zabbix/module.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/pybind/mgr/zabbix/module.py b/src/pybind/mgr/zabbix/module.py
index 638b688562f..2e348ab0391 100644
--- a/src/pybind/mgr/zabbix/module.py
+++ b/src/pybind/mgr/zabbix/module.py
@@ -131,7 +131,7 @@ class Module(MgrModule):
         self._zabbix_hosts = list()
         servers = cast(str, self.config['zabbix_host']).split(",")
         for server in servers:
-            uri = re.match("(?:(?:\[?)([a-z0-9-\.]+|[a-f0-9:\.]+)(?:\]?))(?:((?::))([0-9]{1,5}))?$", server)
+            uri = re.match(r"(?:(?:\[?)([a-z0-9-\.]+|[a-f0-9:\.]+)(?:\]?))(?:((?::))([0-9]{1,5}))?$", server)
             if uri:
                 zabbix_host, sep, opt_zabbix_port = uri.groups()
                 if sep == ':':
